core/tee/tee_rpmb_fs.c

```diff
@@ -267,6 +268,7 @@ struct tee_rpmb_ctx {
 	bool key_derived;
 	bool key_verified;
 	bool dev_info_synced;
+	enum thread_shm_type shm_type;
```

Comment:
This field is only ever set to `THREAD_SHM_TYPE_APPLICATION` in this commit. Do you expect other values can be used later?

Reply:
Yes, it's updated in "core: rpmb: probe for kernel RPMB driver" based on the result of OPTEE_RPC_CMD_RPMB_PROBE_RESET.